jest.config.coverage.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
As discussed, not sure we need this file as we have a similar one already pushed.